test/unit/imgix-provider.test.jsx

```diff
@@ -17,43 +17,60 @@ describe("ImgixProvider", () => {
     const wrappedComponent = (
       <ImgixProvider>
         <ReactImgix {...imageProps} />
+        <Background {...imageProps} />
       </ImgixProvider>
     );
 
+    const renderedComponent = shallow(wrappedComponent);
+
+    // Inspect the rendered children directly
+    const renderedChildren = renderedComponent.children();
+
     const expectedProps = {
-      children: (
-        <ReactImgix
-          src="https://assets.imgix.net/examples/pione.jpg"
-          sizes="50vw"
-        />
-      ),
       value: {},
     };
 
-    const renderedComponent = shallow(wrappedComponent);
-    expect(renderedComponent.props()).toEqual(expectedProps);
+    expect(renderedChildren.length).toBe(2); // Assuming you have two children
+    expect(renderedChildren.at(0).props()).toEqual({
+      src: 'https://assets.imgix.net/examples/pione.jpg',
```

Comment:
suggestion(non-blocking): might as well use `imageProps.src/sizes` here since that is what it should be set to.